### PR TITLE
fix card component

### PR DESCRIPTION
**Pull request by johngruen**

### CODE REVIEW

**Comment by lmorchard** [comment]
# GitHub Pages Deployment
Storybook:
https://FirefoxUX.github.io/photon-components-web/commits/6636505431fb341bdde2469ef657b02103399a90/
Build:
https://FirefoxUX.github.io/photon-components-web/commits/6636505431fb341bdde2469ef657b02103399a90/dist.zip

File: lib/stories-common.js [hunk 1]
```diff
@@ -9,8 +9,8 @@ export const actionDefault = msg => ev => {
 
 export const Sample = ({ children, ...props }) => (
   <ExtractSource {...props}>
-    <div className="sample" onClick={actionDefault("clicked")}>
+    <React.Fragment onClick={actionDefault("clicked")}>
```
Comment (lmorchard) [inline lib/stories-common.js]: Hmm, I don't think this onClick handler works on a React.Fragment. The handler is here to intercept clicks on links and buttons that would jump us out of the Storybook, and it looks like that stops working in the Storybook build on this PR

**Comment by lmorchard** [comment]
# GitHub Pages Deployment
Storybook:
https://FirefoxUX.github.io/photon-components-web/commits/cb08244e606ae718288c7eebbccb29acbee34ef0/
Build:
https://FirefoxUX.github.io/photon-components-web/commits/cb08244e606ae718288c7eebbccb29acbee34ef0/dist.zip

File: lib/stories-common.js (hunk 1 above)
Comment (meandavejustice) [reply lib/stories-common.js]: @lmorchard switched to div and updated `preview-styles.css`

**Comment by lmorchard** [comment]
# GitHub Pages Deployment
Storybook:
https://FirefoxUX.github.io/photon-components-web/commits/6d342f46f0c2678805ced769bd5e9638d8538a1d/
Build:
https://FirefoxUX.github.io/photon-components-web/commits/6d342f46f0c2678805ced769bd5e9638d8538a1d/dist.zip

**Comment by meandavejustice** [comment]
@lmorchard Should be good to go now, I'll let you merge since I made changes

**Pull request merged**